src/main/java/com/google/codeu/data/User.java

```diff
@@ -1,13 +1,24 @@
 package com.google.codeu.data;
 
+//Provides the capability to save the data the user submits, 
+//and then retrieve that data to display the user's "about me" 
+//section on their user page.
+
```

Comment:
Also make sure there's not an extra space between the end of the comment and the beginning of your class.